src/SpectralGaussianSimulation.jl

```diff
@@ -41,6 +41,10 @@ Fourier transform method.*
 end
 
 function preprocess(problem::SimulationProblem, solver::SpecGaussSim)
+  hasdata(problem) && @warn (
+    "Conditional spectral Gaussian simulation is not currently supported. " *
```

Comment:
The following warning would be enough:

```
hasdata(problem) && @warn "Conditional spectral Gaussian simulation is not currently supported"
```

Could you please update the message?

Reply:
Yes, good point.

Reply:
Awesome, thanks! Just a heads up so that you understand better some upcoming development directions. The "hasdata" function will go away in the future, or at least will be refined to accommodate more detailed queries like "hasdata(p, var)". This discussion is also relevant in the presence of missing values for which we have an issue open.